browser/src/core/Socket.js

```diff
@@ -1409,20 +1402,21 @@ app.definitions.Socket = L.Class.extend({
 		}
 		else if (this._reconnecting) {
 			// we are reconnecting ...
-			this._reconnecting = false;
-			this._map._docLayer._resetClientVisArea();
-			this._map._docLayer._requestNewTiles();
+			this._map._docLayer._refreshTilesInBackground();
 			this._map.fire('statusindicator', {statusType: 'reconnected'});
-			this._map._isNotebookbarLoadedOnCore = false;
```

Comment:
This breaks notebookbar after reconnect. It requires to be inicialized properly in the core so our widgets like "style previews" will work.

Reply:
Done thanks